app/src/main/java/com/example/grassroots/CivicInfoAdapter.java

```diff
@@ -0,0 +1,37 @@
+package com.example.grassroots;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.grassroots.model.ElectedRepresentatives;
+
+import java.util.List;
+
+public class CivicInfoAdapter extends RecyclerView.Adapter<CivicInfoViewHolder> {
+
+    private List<ElectedRepresentatives> electedRepresentatives;
+
+    public CivicInfoAdapter(List<ElectedRepresentatives> electedRepresentatives) {
```

Comment:
In general you do not want to take in a list through the constructor of an adapter. This is because you do not want to recreate an object anytime your data changes.